tests/plugins/pastefromword/cleanword.html

```diff
@@ -0,0 +1,41 @@
+<textarea id="root-image">
```

Comment:
I'd rename this test suite simply to `pasteimage` - shorter name, and still descriptive enough.